Maple2.Database/Model/Quest.cs

```diff
@@ -1,47 +1,87 @@
 ﻿using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
+using Maple2.Database.Extensions;
 using Maple2.Model.Enum;
+using Maple2.Model.Metadata;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
 
 namespace Maple2.Database.Model;
 
 public class Quest {
     public int Id { get; set; }
+    public long OwnerId { get; set; }
     public QuestState State { get; set; }
     public int CompletionCount { get; set; }
     public long StartTime { get; set; }
     public long EndTime { get; set; }
-    public long ExpiryTime { get; set; }
-    public bool Accepted { get; set; }
-    public IList<int> Conditions { get; set; }
+    public bool Track { get; set; }
+    public SortedDictionary<int, QuestCondition> Conditions { get; set; }
 
     public Quest() {
-        Conditions = new List<int>();
+        Conditions = new SortedDictionary<int, QuestCondition>();
```

Comment:
Do you know if construction here is needed? In a lot of other cases we don't initialize but it seems EFCore will do it for you.